### ISSUE TITLE
Visual bug

### ISSUE THREAD

**Issue by Hardeh** [cmooref17/ReservedItemSlotMods#7]
When we unlocked both flashligh and walkie slot and put respective items, these items shown simultaneously in player hands all the time
https://i.imgur.com/tYWgmjQ.png

**Comment by cmooref17** [comment]
So you see this on other players and not yourself? Does your friend see two items in your hands? Who is the host?
And thanks for bringing this up!

**Comment by Phyore** [comment]
I get the same issue. It didn't seem to be consistent, some had it on their chest etc, some in the hands, some mixed

**Comment by Hardeh** [comment]
> So you see this on other players and not yourself? Does your friend see two items in your hands? Who is the host? And thanks for bringing this up!
Yes, on other players, and friends see two items in my hands as well. I'm host.
Moreso, seems like when anyone picks up any item from the ground when reserved slots are active - this item stays on the ground for other players, but desynced and impossible to pickup.

**Comment by Phyore** [comment]
> > ... Moreso, seems like when anyone picks up any item from the ground when reserved slots are active - this item stays on the ground for other players, but desynced and impossible to pickup, and if player drops that item - it just teleports to new location for every other player. This is not happening with old version of reserved slots.

Can confirm about the desync glitch, makes it really hard to play. Only this (and the two mods walkie/flash) had updates since last play. There was also an issue where the terminal typing was not displayed on the screen. The stream below can show the various bugs, nearer the end.

I will say I was on 2.0.2
[Stream](https://www.twitch.tv/videos/2089795950)

**Comment by cmooref17** [comment]
With the de-sync glitch, does everyone have the mod, and are on the same version?

**Comment by cmooref17** [comment]
I apologize for my late replies!

**Comment by Hardeh** [comment]
> With the de-sync glitch, does everyone have the mod, and are on the same version?

Yes, synced profile by code

**Comment by cmooref17** [comment]
These issues should be fixed now. Could you let me know if not? Thanks!
